composer.json

```diff
@@ -56,7 +56,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-develop": "1.x-dev"
+            "dev-develop": "2.x-dev"
```

Comment:
Is niet noodzakelijk toch?

Reply:
Ik heb dus wel strict types hier toegevoegd (dit kan eventueel iets breken?). Heb de rest ook nagelopen en de return type van `PlainSmtpTransportFactory::create` is ook aangepast. Die klopte niet met de documentatie die erboven stond (dus in theorie niet BC). 

Anyways; als we er niet voor kiezen om hier een major release van te maken zal ik de alias aanpassen. Anders is het wel handig een 1.x branch te introduceren.